Content.Server/Chat/Systems/ChatSystem.cs

```diff
@@ -491,6 +503,27 @@ private void ClientDistanceToList(EntityUid source, int voiceRange, List<ICommon
         }
     }
 
+    private void ListenerEntityDistanceToList(EntityUid source, List<EntityUid> listenerList)
```

Comment:
Could you possibly convert this into something that returns an `IEnumerable<EntityUid>`? This method works fine enough, otherwise - it's more of a matter of style if anything, as you use the list passed in to just iterate over. Just in case, look up the `yield` keyword for C# for reference.